liquidator/liquidator.js

```diff
@@ -26,8 +40,22 @@ class Liquidator {
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
   queryAndLiquidate = async priceFunction => {
+    const { toBN, fromWei } = this.web3.utils;
+
     const contractTime = this.empClient.getLastUpdateTime();
-    const priceFeed = priceFunction(contractTime);
+    let priceFeed = priceFunction(contractTime);
+
+    // The `priceFeed` is a Number that is used to determine if a position is liquidatable. The higher the
+    // `priceFeed` value, the more collateral that the position is required to have to be correctly collateralized.
+    // Therefore, we add a buffer by reducing `priceFeed` to (`crThreshold` * `priceFeed`).
+    priceFeed = fromWei(toBN(priceFeed).mul(toBN(this.crThreshold)));
```

Comment:
Nit1: numerically this works great but the wording is a bit strange. After multiplying the output of `priceFunction(contractTime)` by `crThreshold` this is not a `priceFeed` any more. Consider re-naming this variable to something else. Perhaps `scaledPriceFeed` or something.

Nit2: I'm not a big fan of re-using this variable like this. it's declared on like 46 and then re-used used in it's own re-calculation. This comment is stylistic more than anything.

Reply:
`priceFeed` should be renamed then to `price`. I was confused at first by this as well. `priceFeed` is a `toWei`'d String.

Reply:
Ah I see your point. I will use `scaledPriceFeed` as that is much clearer